src/client/Microsoft.Identity.Client/IntuneAppProtectionPolicyRequiredException.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// This exception is thrown when Intune requires App protection policy.
+    /// Properties in it can be used by app to obtain the required enrollmentID from MAM SDK
+    /// https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Exceptions/#IntuneAppProtectionPolicyRequiredException
```

Comment:
```suggestion
    /// This exception is thrown when Intune requires app protection policy.
    /// The information in this exception can be used by app to obtain the required 
    /// enrollment ID from Mobile App Management (MAM) SDK.
    /// See https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Exceptions#IntuneAppProtectionPolicyRequiredException
```

Reply:
The wiki page should have some links to the general docs on what MAM is. Also wonder if aka.ms link would be beneficial.

Reply:
I was debating between aka.ms vs github link. Since this is a small topic (just one exception), it may not be worth aka.ms. If a doc/example is done, it may be worth adding later.

Reply:
aka.ms is more flexible. We are trying to move away from wiki pages, because customers like official docs instead.

Reply:
Done. https://aka.ms/msal-net-trueMAM